library/kani/src/tuple.rs

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+// Modifications Copyright Kani Contributors
+// See GitHub history for details.
+
+//! Support for arbitrary tuples where each element implements
+//! `kani::Arbitrary`. Tuples of size up to 11 are supported in this
```

Comment:
Isn't this supporting up to 12 (A - L)?

Reply:
72a48f243631eef863686f1726db395a867df4a0, also fixed PR title so the squash commit says 12.